Polyhedron/demo/Polyhedron/Scene_surface_mesh_item.cpp

```diff
@@ -1505,10 +1505,9 @@ bool
 Scene_surface_mesh_item::save(std::ostream& out) const
 {
   QApplication::setOverrideCursor(Qt::WaitCursor);
-  out.precision(17);
-    out << *(d->smesh_);
-    QApplication::restoreOverrideCursor();
-    return (bool) out;
+  CGAL::write_OFF(out, *(d->smesh_), CGAL::parameters::stream_precision(17));
+  QApplication::restoreOverrideCursor();
+  return (bool) out;
```

Comment:
I find strange that `write_OFF` does not use the precision of the stream. Where is it documented?

Reply:
I think it is because of the default value of the named parameter. It is documented as 6

Reply:
I guess that when the target is a stream, the default value of the NP could have been the precision of the stream.

Reply:
it indeed seems more natural (`CGAL::parameters::is_default_parameter(np)` should be used to detect that)